### ISSUE TITLE
Arrow Glitch

### ISSUE THREAD

**Issue by imryaan** [Avicus/Issues#1204]
Recently, I have been hearing from players that their arrows are bouncing back when fired. Making it look like they're being blocked by a spectator.

**Comment by ghost** [comment]
They look bounced back, but they still reach their target. I think it's due to the multiple versions.

**Comment by ghost** [comment]
If you look closely, you can see it bounce then a few blocks farther bounce again.

**Comment by Palmidence** [comment]
Even though it looks like they might go down or bounce back, they're still travelling like usual. Tested.

**Comment by master00ofdsiware** [comment]
This is still a glitch. Arrows should not be bouncing, even if it's only a visual effect.

**Comment by Palmidence** [comment]
It's a client side issue. There's nothing we can do to fix visual effects. There are other client side issues that exist, like lilypads or doors, but we can't fix them because the client handles it, not the server.

**Comment by master00ofdsiware** [comment]
I will test this myself and open a new issue if I believe it is necessary.
